content/3_Silver/Intro_Bitwise.mdx

```diff
@@ -349,24 +349,11 @@ The special property that differentiates it from the other bitwise operations is
 
 ### Explanation
 
-Simulating the process, similarly to [Pascal's triangle](https://en.wikipedia.org/wiki/Pascal%27s_triangle), isn't efficient,
-because of $n$ being too large. One good practice when dealing with such problems is analyzing how the answer is affected by each single value. Keeping in mind that xor is its
-own inverse, a value $x$ xored with itself an **even** number of times will result in $0$, thus canceling each other out; on the other hand, the value $x$ xored
-with itself and **odd** number of times will simply result in $x$. Consequently, we can shift our focus to finding for each bottom value the number of times it will affect
-in the top value.
-
-Let's take a look at how bottom values modify the final result.
-
-![pyramid](./assets/xor-pyramid.png)
-
-As you can see, the value $B$ does not appear in the final result, thus it can be ignored. Additionally, the bottom row values in
-Pascal's triangle of height five are $1,4,6,4,1$. We can think of these values as the number of occurrences of each bottom row value.
-In the above image, value $A$ will be xored one time in the result, while value $B$ will be xored four times in the result. Therefore,
-the parity of the frequency tells us that value $A$ contributes to the result, while value $B$ cancels out itself.
-
-Because we are only interested in the parity of the binomial coefficient, there is no need to compute its actual value. We can simply check the parity by
-counting the power of two in $\binom{n}{k}=\frac{n!}{k! \cdot (n-k)!}$. With this in mind, precompute $pref_i=p$ where $p$ is the power of two in $i!$ factorization.
-Therefore, the binomial coefficient is odd if: $pref_n-pref_k-pref_{n-k}=0$.
+Let's analyze the bits of each xor-sum independently. In this case, we'll check for every bit position, i.e.
+for every power of two, whether or not it's set in every contiguous subsequence xor-sum.
```

Comment:
> contiguous subsequence xor-sum

do you mean the xor-sum of every contiguous subsequence?

Reply:
yes

Reply:
I rephrased it